Falende links check op TCP `connection refused` error

Dit gebeurt op verscheidene repositories met verschillende URLs die niet werken. Voorbeeld failures:

https://github.com/Logius-standaarden/API-Design-Rules/actions/runs/13180942174/job/36792576813
https://github.com/Logius-standaarden/logboek-dataverwerkingen/actions/runs/13243164499/job/36963106083

In beide gevallen faalt het op het opvragen van een document uit de repository zelf (dat niet de `snapshot.html` is):

```
http://localhost:8080/snapshot.html
	error when dialing [::1]:8080: dial tcp [::1]:8080: connect: connection refused	http://localhost:8080/media/style.css
Error: Process completed with exit code 1.
```

Op Stackoverflow staan enkele hints die er op wijzen dat de Python server mogelijk in een aparte side-container moet worden gedraaid middels de `services` setting: https://stackoverflow.com/a/74341376 met bijbehorende GitHub documentatie: https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/about-service-containers

--- a/.github/workflows/check.yml
+++ b/.github/workflows/check.yml
@@ -27,8 +27,23 @@ jobs:
           path: ~/static
           key: ${{ github.run_id }}
       - run: mv ~/static/snapshot.html snapshot.html
-      - name: Local server
-        run: python3 -m http.server 8080 &
+      - name: Serve Files
+        uses: Eun/http-server-action@v1
+        with:
+          content-types: |
+            {
+              "css": "text/css",
+              "html": "text/html",
+              "ico": "image/x-icon",
+              "jpeg": "image/jpeg",
+              "jpg": "image/jpeg",
+              "js": "text/javascript",
+              "json": "application/json",
+              "png": "image/png",
+              "svg": "image/svg+xml",
+              "txt": "text/plain",
+              "xml": "text/xml"
+            }
       - run: echo '/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin' >> $GITHUB_PATH
       - name: Install muffet
         run: brew install muffet      

